underactuated.html

```diff
@@ -1312,20 +1329,65 @@ <h1>The Simple Pendulum</h1>
 
   <section><h1>Exercises</h1>
 
-    <exercise><h1></h1>
+    <exercise><h1>Graphical Analysis</h1>
+
+      <figure>
+      <img width="60%" data-src="figures/exercises/graphical_analysis.svg"/>
+      <figcaption>First-order systems with multiple equilibria.</figcaption>
+      </figure>
+
+      Consider the first-order system $$\dot x = \begin{cases} - x^5 + 2 x^3 - x & \text{if} \quad x \leq 1, \\ 0 & \text{if} \quad 1 < x \leq 2, \\ - x + 2 & \text{if} \quad x > 2, \end{cases}$$ whose dynamics is represented in the figure above.  Notice that, together with $x^*=-1$ and $x^*=0$, all the points in the closed interval $[1,2]$ are equilibria for this system.  For each equilibrium point, determine whether it is stable i.s.L., asymptotically stable, and/or exponentially stable.
+
+    </exercise>
+
+    <exercise><h1>Basin of Attraction and Bifurcation Diagram</h1>
 
       Consider the first-order system with polynomial dynamics $$\dot x = f(x) = - x^3 + 4x^2 +11x - 30.$$
 
       <ol type="a">
 
         <li> Sketch the graph of the function $f(x)$, and identify all the equilibrium points.</li>
 
-        <li> For each equilibrium point, determine whether it is stable or unstable.  For each one of the stable equilibria, identify the region of attraction.</li>
+        <li> For each equilibrium point, determine whether it is stable or unstable.  For each one of the stable equilibria, identify the basin of attraction.</li>
 
         <li> Consider an additive term $w$ in the system dynamics: $\dot x = f(x) + w$.  As $w$ ranges from $0$ to $\infty$, determine how many stable and unstable equilibrium points the system has. Support your answer with a sketch of the bifurcation diagram for nonnegative values of $w$.</li>
 
       </ol>
 
+      (A note on the terminology: when there is no need to distinguish between different notions of stability, we simply classify equilibrium points as "stable" or "unstable.")
```

Comment:
if a point is i.S.L but not exponentially stable, would you say "stable" or "unstable".  your note is ambiguous on that point.

Reply:
I meant stable in all senses (i.s.L., asymptotically, and exponentially). This is something that makes me frequently think: when writing, it's so convenient to just say "stable" without specifying all the times what I'm referring to. However, when I say stable, in general, I mean both i.s.L. and asymptotically but not necessarily exponentially. E.g., I'd define xdot=-x^3 stable. This is somehow standard, right? Is it worth specifying it in the textbook?